throneless-tech/recoding-tech
# Updates to CMS!

With this PR comes a whole host of small but significant changes to the Recoding.tech Sanity backend.  

In particular, this pull request makes the following changes:
- [x] Rename `policy` into `policy action` and `syllabus` to `quick start guide`
- [x] Enable Portable Text/rich text editor in the `Policy action` model and `Quick start guide` page
- [x] The `Quick start guide` rich text editor also enables embedding a reference to a `citation` inline  
- [x] Each `topic` has an option to refer to a `quick start guide` 

To test this, you should:
`npm install` and make sure your .env file is up to date (ping me if you need some credentials!), `npm run dev`, and `sanity start` in the `studio` folder

P.S. I love Brisket the dog.
